src/test/csf/submitters.h

```diff
@@ -58,13 +58,13 @@ struct Rate
             arithmetic T to SimDuration::rep units to allow using standard
             library distributions as a Distribution.
 */
-template <class Distribution, class Generator>
+template <class Distribution, class Generator, class Selector>
 class Submitter
 {
     Distribution dist_;
     SimTime stop_;
     std::uint32_t nextID_ = 0;
-    Peer & target_;
+    Selector & selector_;
```

Comment:
No explicit motivation. Just didn't want to deal with working out the possible semantic issues as if `Selector` accidentally became not trivially copyable as I cludged things to work. In particular I was caught up in unrelated changes to `Selector` to allow it to be used with arbitrary `iterator`s to `T` instead of only taking a vector. As I unraveled those changes because I couldn't get them to work properly I simply forgot about using `Selector &`.